src/lib/server/ApiClient.ts

```diff
@@ -1,11 +1,48 @@
-export type UploadRequest = {
+export type UploadRequestResponse = {
     ObjectId: number,
     ObjectKey: string,
-    StatusName: string
+    StatusName: string,
+    SizeBytes: number,
+    TakenAtSec: number
 }
 
-export async function getUploadRequest(): Promise<UploadRequest[]> {
+export type PostInsertRequestRequest = {
```

Comment:
We don't have `InsertRequest` domain thingy. Let's use `UploadRequest` to be consistent.